.github/workflows/benchmark.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           key: ${{ runner.os }}-${{ steps.node.outputs.v8CppApiVersion }}-${{ hashFiles('**/yarn.lock', '**/package.json') }}
       - name: Install & build
         if: steps.cache-deps.outputs.cache-hit != 'true'
-        run: yarn install --frozen-lockfile --ignore-optional && yarn build
+        run: yarn install --frozen-lockfile --ignore-optional --ignore-scripts && yarn build
```

Comment:
Does this flag `--ignore-scripts` ignore all scripts? Or gets intercepted by lavamoat somehow to only apply to a subset of scripts?

Reply:
It ignores all scripts. I updated the description of the PR to give an overview of how lavamoat works